docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+    "\n",
+    "At this point we are ready to run AFQMC. We need to build an `AFQMC` driver class which takes as input the factorized Hamiltonian and our `ParticleHole` trial wavefunction."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipie.qmc.afqmc import AFQMC\n",
+    "\n",
+    "qmc_driver = AFQMC.build(num_elec, ham, wfn, num_blocks=300, num_walkers=50, seed=7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.config import OUTDIRS\n",
+    "import pathlib\n",
+    "ipie_path = pathlib.Path(pyscf_hamiltonian.params.path_prefix + OUTDIRS.DEFAULT_QMC_DIRECTORY)\n",
+    "if not ipie_path.is_dir():\n",
+    "    ipie_path.mkdir(parents=True)\n",
+    "qmc_driver.run(estimator_filename=f'{ipie_path}/estimates.h5')"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we can visualize the data which is by default saved to `estimates.0.h5`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipie.analysis.extraction import extract_observable\n",
+    "import matplotlib.pyplot as plt\n",
+    "data = extract_observable(qmc_driver.estimators.filename)\n",
+    "plt.plot(data.ETotal, marker='o', label=\"AFQMC\", color=\"C0\")\n",
+    "plt.axhline(-1.969512, label=\"Exact Result\", color=\"C1\")\n",
+    "plt.legend()\n",
+    "plt.xlabel(\"Block number\")\n",
+    "plt.ylabel(\"Total Energy (Ha)\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Through visual inspection we can determine when the calculation has equilibrated, after which we should be sampling the (approximate) ground state. We can use samples from this point on to estimate the AFQMC energy and error bar. The function `reblock_minimal` will perform the necessary error analysis taking into account the serial temporal correlation in the AFQMC data. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ipie.analysis.blocking import reblock_minimal \n",
+    "reblock_minimal(qmc_driver.estimators.filename, start_block=20)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that the number of walkers and the number of blocks is probably too low to obtain statistically significant results and we would advise increasing both of these. For larger scale simulations would can use MPI to distribute the work among many MPI processes. See [ipie](https://github.com/JoonhoLee-Group/ipie/) for further details."
```

Comment:
Comma after "For larger scale simulations".  Also, the following word is not correct.  Did you mean "we can use MPI to ..."

Reply:
Done.